akid/akids.go

```diff
@@ -121,6 +121,10 @@ func (APISpecID) GetType() string {
 	return APISpecTag
 }
 
+func (id APISpecID) String() string {
+	return String(id)
```

Comment:
Seeing the unqualified function name everywhere was super-confusing.  :)  Nothing to fix here, though.

Reply:
> Could be resolved by having the akid "type" be a member of the base class instead of retrieved via a function, but that would be a lot of churn too.

Yeah, this occurred to me too, but I decided against it—there's something satisfying about having AKIDs be represented by just UUIDs.